pywhat/Data/regex.json

```diff
@@ -163,6 +163,171 @@
           "Networking"
        ]
     },
+    {
+      "Name": "Google API Key",
+      "Regex": "AIza[0-9A-Za-z-_]{35}",
+      "plural_name": false,
+      "Description": "API Key for accessing Google services including [#CAE4F1][link=https://developers.google.com/maps/documentation/javascript/get-api-key]Google Maps[/link][/#CAE4F1]",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
+         "API Keys",
+         "Credentials",
+         "Google"
+      ]
+    },
+    {
+      "Name": "Google ReCaptcha API Key",
+      "Regex": "6L[0-9A-Za-z-_]{38}|^6[0-9a-zA-Z_-]{39}$",
+      "plural_name": false,
+      "Description": "[#CAE4F1][link=https://developers.google.com/recaptcha/intro]API Key for Google ReCaptcha[/link][/#CAE4F1]",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
+         "API Keys",
+         "Credentials",
+         "Google"
+      ]
+   },
+   {
+      "Name": "Google OAuth Token",
+      "Regex": "ya29.[0-9A-Za-z-_]+",
+      "plural_name": false,
+      "Description": "[#CAE4F1][link=https://developers.google.com/identity/protocols/oauth2]Token for Google OAuth[/link][/#CAE4F1]",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
+         "API Keys",
+         "Credentials",
+         "Google"
+      ]
+   },
+   {
+      "Name": "AWS Access Key ID",
+      "Regex": "A[SK]IA[0-9A-Z]{16}",
+      "plural_name": false,
+      "Description": "[#CAE4F1][link=https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html]Access Key ID for AWS Credentials[/link][/#CAE4F1]",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
+         "API Keys",
+         "Credentials",
+         "AWS"
+      ]
+   },
+    {
+      "Name": "Mailgun API Key",
+      "Regex": "key-[0-9a-zA-Z]{32}",
+      "plural_name": false,
+      "Description": "[#CAE4F1][link=https://documentation.mailgun.com/en/latest/api-intro.html#introduction]API Key for Mailgun[/link][/#CAE4F1]",

```

Comment:
Hey! What do you think about getting rid of these descriptions? I believe they're the same as the name, just with a link attached? 😄

Reply:
Yeah that seems reasonable!